repository/repository-web/dist/partials/admin/modeshapeDoctor.html

```diff
@@ -0,0 +1,112 @@
+<h1>Modeshape Doctor</h1>
+<div>
+    <h2 style="color: red"><i class="fa fa-exclamation-triangle"></i>Warning! Use the write/delete functions with caution!<i class="fa fa-exclamation-triangle"></i></h2>
+    <h4>Known limitations:</h4>
+    <ul>
+        <li>Writing values on multi-value properties is not supported.</li>
+    </ul>
+</div>
+<table class="table table-condensed">
+    <tr>
+        <td class="col-md-3"><label>Workspace ID:</label></td>
+        <td class="col-md-7"><input type="text" ng-model="modeshapeWorkspaceId" style="width: 100%"></td>
```

Comment:
Would it be possible to have some search functionality by namespace as an alternative? It's true that the ModeShape doctor is inherently for sysadmins who are expected to be able to query the namespace table for that workspace ID, but it would be much nicer IMO to have that straight away.